test/tests/functional/basic_single_server_auth_tests.js

```diff
@@ -8,7 +8,8 @@ var expect = require('chai').expect,
   Bson = require('bson');
 
 describe('Basic single server auth tests', function() {
```

Comment:
Is every test `skip`ed in this `describe` block? We could just do `describe.skip`

Reply:
yeah I'm not sure what happened here, I think these were skipped from a while ago but sure I can `describe.skip` it.